swagger.yaml

```diff
@@ -275,18 +448,13 @@ definitions:
       next:
         type: string
         description: Name of the API.
-      time:
-        type: string
-        description: Build time in ISO 8601 format.
-      unixTime:
-        type: integer
-        description: Build time as milliseconds since the unix epoch.
-      commit:
+
+  ServiceLinks:
+    properties:
+      self:
         type: string
-        description: Git commit hash of the revision which was built.
-      documentation:
+      location:
```

Comment:
I don't think we should have this in the links section. I would recommend having the location link in the relationship section similar to how location objects link to services. Also, json api spec only allows links objects to contain "self" and "related" links. You could have the api respond with something like:
```
"relationships": {
    "location": {
        "links":{
            "related": "api.o.e/locations/idofparentlocation"
        }
    }
}